docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -405,6 +405,9 @@ The same path should be used for `MISO_FILES_DIR` as is set for `miso.fileStorag
 jdbc:mysql://localhost:3306/lims?autoReconnect=true&zeroDateTimeBehavior=convertToNull&useUnicode=true&characterEncoding=UTF-8
 ```
 
+If you encounter errors migrating the database, make sure that you are using the recommended version of Flyway (see above).
+There are known issues with using newer Flyway versions with the MISO migrations.
```

Comment:
Might be worth saying something like "If migration fails on V000#, you may be using a newer version of Flyway which MISO does not support. As Flyway has changed their checksum calculations in newer versions and we don't want to break completed migrations for existing users, there are no plans to support newer versions of Flyway." (or a similar explainer of why we won't be supporting a newer version of Flyway)

Reply:
I don't want to mention specific errors or locations of errors since they may vary depending on the exact Flyway version. If _anything_ fails, step 1 should be to make sure you're using the correct version.

I'll add the note about why we don't plan on supporting newer versions though.